releases/v21.2.0.md

```diff
@@ -0,0 +1,127 @@
+Core       | **Transaction guardrails**     | Transaction guardrails have been added to CockroachDB to improve production stability. These can help prevent cluster unavailability and protect the cluster against multiple developer workloads. <!-- add link when Stephanie publishes PR: ../v21.2/transactions.html#limit-the-number-of-rows-written-or-read-in-a-transaction -->
+Core       | **Expression indexes**         | [Indexes on expressions](../v21.2/expression-indexes.html) can now be created. These indexes speed up queries that filter on the result of that expression, and are especially useful for indexing only a specific field of a `JSON` object.
+Core       | **Correlated CTEs**            | [Correlated common table expressions](../v21.2/common-table-expressions.html#correlated-common-table-expressions) (CTEs) are now supported in CockroachDB. A correlated CTE is when a common table expression is contained in a subquery and the CTE references columns defined outside of the subquery.
+Core       | **Persistent statement and transaction statistics** | Statistics information on the [**Statements**](../v21.2/ui-statements-page.html) and [**Transactions**](../v21.2/ui-transactions-page.html) pages within the DB Console can now be persisted for longer than one hour.
+Core       | **Restricted and default placement** | You can now use the [`ALTER DATABASE ... PLACEMENT RESTRICTED`](../v21.2/placement-restricted.html) statement to constrain the replica placement for a [multi-region database](../v21.2/multiregion-overview.html)'s [regional tables](../v21.2/regional-tables.html) to the [home regions](../v21.2/set-locality.html#crdb_region) associated with those tables.
+Core       | **Bounded staleness reads**    | [Bounded staleness reads](../v21.2/follower-reads.html#bounded-staleness-reads) are now available in CockroachDB. These use a dynamic, system-determined timestamp to minimize staleness while being more tolerant to replication lag than exact staleness reads. This dynamic timestamp is returned by the `with_min_timestamp()` or `with_max_staleness()` [functions](../v21.2/functions-and-operators.html). In addition, bounded staleness reads provide the ability to serve reads from local replicas even in the presence of network partitions or other failures.
+Core       | **Automatic ballast files**    | CockroachDB now automatically creates an emergency ballast file at startup time.  The `cockroach debug ballast` command is still available but deprecated. For more information about how automatic ballast file creation works, see [automatic ballast files](../v21.2/cluster-setup-troubleshooting.html#automatic-ballast-files).
+
```

Comment:
Contention views

You can now use pre-built contention views in [crdb_internal](https://www.cockroachlabs.com/docs/dev/crdb-internal.html#data-exposed-by-crdb_internal) to quickly identify the top contending indexes. These views can be used to [understand where and avoid contention](https://www.cockroachlabs.com/docs/v21.1/performance-best-practices-overview#understanding-and-avoiding-transaction-contention) happening in your workload.